Resources/doc/reference/search.rst

```diff
@@ -55,6 +55,30 @@ You can also configure the block template per admin while defining the admin:
               </call>
           </service>
 
+
+
+Configure the default search result action.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In general the search result generates a link to the edit action of an item or is using the show action, if the edit
+route is disabled or you haven't the required permission. You can change this behaviour by overriding the
+``searchResultActions`` property. The defined action list will we checked successive until a route with the required
+permissions exists. If no route is found, the item will be displayed as a text.
+
+.. code-block:: php
+
+    <?php
+    // src/AppBundle/Admin/PersonAdmin.php
+
+    class PersonAdmin extends Admin
+    {
+        // ...
+
+        protected $searchResultActions = array('edit', 'show');
```

Comment:
We use protected properties for some stuff in the ``Admin`` class, so we should continue with this. 

Feel free to create an issue for the 4.x branch where we could discuss getters vs. properties @OskarStark